.scripts/semaphore.js

```diff
@@ -1,103 +1,100 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 // original sources can be found at https://github.com/abrkn/semaphore.js
```

Comment:
Does this license notice still make sense given how much the impl has changed?

Reply:
Yeah I think so. The impl itself hasn't changed. It's just been converted to common module format.